src/main/java/at/hannibal2/skyhanni/config/features/garden/laneswitch/LaneswitchNotificationConfig.java

```diff
@@ -0,0 +1,24 @@
+package at.hannibal2.skyhanni.config.features.garden.laneswitch;
+
+import at.hannibal2.skyhanni.config.FeatureToggle;
```

Comment:
Please cleanup imports before committing changes. When using IntelliJ, you can even automate this:
![image](https://github.com/hannibal002/SkyHanni/assets/24389977/33324e1b-1c3c-4e83-aba1-6cf60bcca2e9)